test_scripts/Polices/External_UCS/External_Consent_ON/001_ATF_Policies_External_Consent_ON_disallowed_by_external_consent_entities_off_data_consent_disallowed.lua

```diff
@@ -1,3 +1,6 @@
+-------------------------------------- Requirement summary -------------------------------------------
```

Comment:
@istoimenova, Please notice, there was made an agreement with management not to fix such issues right now and to move this task to backlog.
Appropriate task was created, with "Fix LuaCheck issues in scripts" summary